src/applications/vaos/express-care/redux/reducer.js

```diff
@@ -109,6 +110,10 @@ export default function expressCareReducer(state = initialState, action) {
         newRequest: {
           ...state.newRequest,
           pageChangeInProgress: false,
+          previousPages:
+            action.direction === 'next'
+              ? state.newRequest.previousPages.concat([action.pageKey])
+              : state.newRequest.previousPages.slice(0, -1),
```

Comment:
I was playing around with this on a review instance and I think I found a case where the array doesn't work. If you use the browser's back button to go back a page, you end up having to click on the "Back" button on the form twice to go back.

I think using the object approach we started with will solve that, rather than trying to do anything like listening for history changes.

Reply:
This makes sense that it'd break 👎

Reply:
i will get working on the object use case